### DISCUSSION TITLE
i have error to bulid example

### DISCUSSION THREAD

**Discussion by almhajer** [ocornut/imgui#5382] [New Users Build/Link/Run issues ONLY!]
hi every one
i learn imgui
i traid to start project small window withe out all other file in mac
i use mac and clion program after run i recive thos error 
can you please help
thanks appy day

===================[ Clean | Debug ]===========================================
"/Applications/CLion 2022.2 EAP.app/Contents/bin/cmake/mac/bin/cmake" --build /Users/mac/CLionProjects/repair/cmake-build-debug --target clean -j 3
[1/1] Cleaning all built files...
Cleaning... 0 files.

Clean finished

====================[ Build | all | Debug ]=====================================
"/Applications/CLion 2022.2 EAP.app/Contents/bin/cmake/mac/bin/cmake" --build /Users/mac/CLionProjects/repair/cmake-build-debug --target all -j 3
[2/2] Linking CXX executable repair
FAILED: repair 
: && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -g -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.3.sdk  CMakeFiles/repair.dir/main.cpp.o -o repair -L/Users/mac/CLionProjects/repair/dep/lib -Wl,-rpath,/Users/mac/CLionProjects/repair/dep/lib  -framework  OpenGL  -framework  OpenGL  ../dep/lib/libglfw.3.3.dylib  -framework  OpenGL  -framework  OpenGL  ../dep/glew/lib/libGLEW.2.2.0.dylib  ../dep/lib/libglfw.3.3.dylib  ../dep/glew/lib/libGLEW.2.2.0.dylib && :
Undefined symbols for architecture x86_64:
  "ImGui_ImplOpenGL3_Init(char const*)", referenced from:
      _main in main.cpp.o
  "ImGui_ImplGlfw_NewFrame()", referenced from:
      _main in main.cpp.o
  "ImGui_ImplGlfw_Shutdown()", referenced from:
      _main in main.cpp.o
  "ImGui_ImplOpenGL3_NewFrame()", referenced from:
      _main in main.cpp.o
  "ImGui_ImplOpenGL3_Shutdown()", referenced from:
      _main in main.cpp.o
  "ImGui_ImplGlfw_InitForOpenGL(GLFWwindow*, bool)", referenced from:
      _main in main.cpp.o
  "ImGui_ImplOpenGL3_RenderDrawData(ImDrawData*)", referenced from:
      _main in main.cpp.o
  "ImGui::ColorEdit3(char const*, float*, int)", referenced from:
      _main in main.cpp.o
  "ImGui::GetDrawData()", referenced from:
      _main in main.cpp.o
  "ImGui::SliderFloat(char const*, float*, float, float, char const*, int)", referenced from:
      _main in main.cpp.o
  "ImGui::CreateContext(ImFontAtlas*)", referenced from:
      _main in main.cpp.o
  "ImGui::DestroyContext(ImGuiContext*)", referenced from:
      _main in main.cpp.o
  "ImGui::ShowDemoWindow(bool*)", referenced from:
      _main in main.cpp.o
  "ImGui::StyleColorsDark(ImGuiStyle*)", referenced from:
      _main in main.cpp.o
  "ImGui::DebugCheckVersionAndDataLayout(char const*, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long)", referenced from:
      _main in main.cpp.o
  "ImGui::End()", referenced from:
      _main in main.cpp.o
  "ImGui::Text(char const*, ...)", referenced from:
      _main in main.cpp.o
  "ImGui::Begin(char const*, bool*, int)", referenced from:
      _main in main.cpp.o
  "ImGui::GetIO()", referenced from:
      _main in main.cpp.o
  "ImGui::Button(char const*, ImVec2 const&)", referenced from:
      _main in main.cpp.o
  "ImGui::Render()", referenced from:
      _main in main.cpp.o
  "ImGui::Checkbox(char const*, bool*)", referenced from:
      _main in main.cpp.o
  "ImGui::NewFrame()", referenced from:
      _main in main.cpp.o
  "ImGui::SameLine(float, float)", referenced from:
      _main in main.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ninja: build stopped: subcommand failed.

**Comment by rokups** [comment]
This is not Dear ImGui question, but programming basics question. Please seek help in appropriate forum. Hint: you do not build Dear ImGui at all.
